Lecture10/Task1/Program.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+
+namespace Task1
+{
+    class Program
+    {
+        static Motocycle StartProgram()
+        {
+            Console.Write("Input model {string}");
+            string inpModel = Console.ReadLine();
+
+            Console.Write("Input proizvoditel {string}");
+            string inpProizvoditel = Console.ReadLine();
+
+            Console.Write("Input probeg {int}");
+            int inpProbeg = int.Parse(Console.ReadLine());
+
+            Console.Write("Input vSize {int}");
+            int intvSize = int.Parse(Console.ReadLine());
+
+            Console.Write("Input power {int}");
+            int intPower = int.Parse(Console.ReadLine());
+
+            Console.Write("Input yearBirth {int}");
+            int dateBirth = int.Parse(Console.ReadLine());
+
+            Motocycle moto = new Motocycle(inpModel, inpProizvoditel, inpProbeg, intvSize, intPower, dateBirth);
+            Random random = new Random();
+            moto.ID = random.Next(1, 100);
+            return moto;
+        }
+        static void Main(string[] args)
+        {
+            Motocycle[] motoArray = new Motocycle[3];
+            motoArray[0] = Program.StartProgram();
+            motoArray[1] = Program.StartProgram();
+            motoArray[2] = Program.StartProgram();
+
+            foreach (var item in motoArray)
+            {
+                Console.WriteLine("Id = " + item.ID + "| Model = " + item.Model + "| Probeg = " + item.Probeg + "| Proizvoditel = " + item.Proizvoditel + "| BirthYear = " + item.BirthYear);
+            }
+        }
+    }
+    class Motocycle
+    {
+        public Motocycle(string inpModel, string inpProizvoditel, int inpPropeg, int intvSize, int intPower, int dateBirth)
+        {
+            Model = inpModel;
+            Proizvoditel = inpProizvoditel;
+            Probeg = inpPropeg;
+            BirthYear = dateBirth;
+            Engine engine = new Engine(intvSize, intPower);
+        }
+        private int _id;
+        public int ID
+        {
+            get { return _id; }
+            //Не работает init. C# < 9.0.
+            set
```

Comment:
possible to make this Prop both read-only or put private modifier - `private set`